IntelliTect.TestTools.WindowsTestWrapper/IntelliTect.TestTools.WindowsTestWrapper/Utilities.cs

```diff
@@ -4,55 +4,49 @@
 
 namespace IntelliTect.TestTools.WindowsTestWrapper
 {
-    public class Utilities
+    public static class Utilities
     {
+        //TODO: Think about moving or renaming this. Handle exception cases better than below.
         /// <summary>
         /// Parse a text file and return a string of its contents
         /// </summary>
         /// <param name="path">The full path of the location and file name</param>
         /// <returns>Returns the text of the chosen file</returns>
-        public string ParseTextFile(string path)
+        public static string ParseTextFile(string path)
         {
             string text = null;
             int retryAttempts = 0;
-            while (retryAttempts < 10)
+            do
             {
-                try
-                {
-                    text = File.ReadAllText(path);
-                }
-                catch (Exception e)
-                {
-                    Console.WriteLine("The file could not be read");
-                    Console.Write(e.Message);
-                }
-
-                if (text != null)
-                {
-                    break;
-                }
-
-                Thread.Sleep(500);
+                text = File.ReadAllText(path);
+                Thread.Sleep( 100 );
                 retryAttempts++;
-            }
+            } while ( retryAttempts < 10 );
             return text;
         }
 
         /// <summary>
         /// Deletes a file given a location
         /// </summary>
         /// <param name="location">The location of the file without a slash at the end</param>
-        /// <param name="filename">The name and extension of the file</param>
-        public void DeleteDocument(string location, string filename)
+        /// <param name="fileName">The name and extension of the file</param>
+        public static void DeleteDocument(string location, string fileName)
         {
-            try
+            int retryAttempts = 0;
+            do
             {
-                File.Delete(Path.Combine(location, filename));
-            }
-            catch (Exception e)
+                File.Delete(Path.Combine(location, fileName));
+                Thread.Sleep( 100 );
+                if (!File.Exists(Path.Combine(location, fileName)))
+                {
+                    break;
+                }
+                retryAttempts++;
+
+            } while ( retryAttempts < 10 );
+            if ( File.Exists(Path.Combine(location, fileName)) )
```

Comment:
Rather than doing the Path.Combine 3 times, just do it once at the beginning of the method and store the result in a local variable.
You can also update the XML docs, since it does not matter if the `location` parameter has the slash at the end.